doc/docs/reference/control-interface.md

```diff
@@ -190,6 +177,8 @@ flowchart TD
     further_processing --> fin
 ```
 
+<figcaption>Read response message via control interface</figcaption>
```

Comment:
I think `<figcaption>` is supposed to be used inside a `<figure>`element. As here it used standalone it is not centered. I have tried to add a `<figure>` element around mermaid and the caption but then the image rendered by mermaid gets very small.

So we have the following options:

1. Have a centered caption but very small images - Not good
2. Have no caption at all - Caption is helpful for some images
3. Replace mermaid with images that are rendered offline - Maybe something for the future.
4. Keep this PR as it is

From all options I think the last one is best for the time being

Reply:
I have added comments to the makrdown sources about it. Is it okay for you?